content/docs/releases/release-notes/release-notes-1.18.md

```diff
@@ -11,26 +11,56 @@ a new default for `Certificate.Spec.PrivateKey.RotationPolicy` now set to `Alway
+### ACME HTTP01 challenge paths now use `PathType` `Exact` in Ingress routes
 
-> 🐛 See [cert-manager/issues/7791](https://github.com/cert-manager/cert-manager/issues/7791).
+> ⚠️ Breaking change
 
-In cert-manager `v1.18.0`, we changed the default `PathType` from `ImplementationSpecific` to `Exact`, in the Ingress routes that are created by the ACME HTTP01 challenge controller.
-This was to support Ingress controllers such as Cilium, which treat `ImplementationSpecific` paths as regular expressions.
+We have changed the `PathType` for ACME HTTP01 Ingress-based challenges to `Exact`.
+This security feature ensures that the challenge path (which is an exact path)
+is not misinterpreted as a regular expression or some other Ingress-specific
+(`ImplementationSpecific`) parsing.
+This allows HTTP01 challenges to be solved when using standards compliant
+Ingress controllers such as Cilium.
 
-But the change is incompatible with certain versions and configurations of the `ingress-nginx` Ingress controller.
+This change is incompatible with certain versions and configurations of the `ingress-nginx` Ingress controller.
 Versions of [`ingress-nginx >=1.8.0`](https://github.com/kubernetes/ingress-nginx/blob/main/changelog/controller-1.8.0.md) support a [`strict-validate-path-type` configuration option](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#strict-validate-path-type) which, when enabled, disallows `.` (dot) in the path value. This is a [bug](https://github.com/kubernetes/ingress-nginx/issues/11176) which makes it impossible to use various legitimate URL paths, including the  `http://<YOUR_DOMAIN>/.well-known/acme-challenge/<TOKEN>` URLs used for [ACME HTTP01](https://letsencrypt.org/docs/challenge-types/#http-01-challenge).
 To make matters worse, the buggy validation is [enabled by default](https://github.com/kubernetes/ingress-nginx/pull/11819) in [`ingress-nginx >= 1.12.0`](https://github.com/kubernetes/ingress-nginx/blob/main/changelog/controller-1.12.0.md).
+You will see errors like this in the cert-manager controller logs:
 
-We are working on a fix. The next cert-manager patch release `v1.18.1` (release date is yet to be decided) will gate the `PathType: Exact` change behind a feature gate, which will be enabled by default. This will allow you to reinstate the old `PathType: ImplementationSpecific` behavior, by disabling the feature gate.
+> Error presenting challenge: admission webhook `validate.nginx.ingress.kubernetes.io` denied the request: ingress contains invalid paths: path `/.well-known/acme-challenge/oTw4h9_WsobTRn5COTSyaiAx3aWn0M7_aYisoz1gXQw` cannot be used with `pathType` Exact
 
-Meanwhile, you have two options:
-1. Do not upgrade cert-manager. Continue to use cert-manager 1.17.
-2. Disable the `strict-validate-path-type` option in your ingress-nginx controller.
+If you use `ingress-nginx`, you have two options:
```

Comment:
Could we somehow make it even clearer that you should do one of the options, not both. I read a bit fast, and first thought a user had to do both changes.